### ISSUE TITLE
Rogue Galaxy - Weird capped FPS when switching HW to SW to HW

### ISSUE THREAD

**Issue by RedDevilus** [PCSX2/pcsx2#3270]
***PCSX2 version:***
v1.5.0-dev-3387-g420f642f6 (1.6.0)

***PCSX2 options:***
![WikiSettings](https://user-images.githubusercontent.com/24227051/75618095-42e1a800-5b69-11ea-8da1-908f68a8b99f.png)

Like on the wiki: 
![WikiComment](https://user-images.githubusercontent.com/24227051/75618110-83d9bc80-5b69-11ea-8f98-a6318f14e7dc.png)

Default Direct 3D 11 HW then switch to SW and then HW while you run around/fight.
If you stay on HW mode it will sets itself to full speed.
Same problem if u load a game 

![LoadGameMemoryCard](https://user-images.githubusercontent.com/24227051/75618167-77099880-5b6a-11ea-9953-6f6bb6356214.png)

But afterwards its fine: 
![Screenshot_652](https://user-images.githubusercontent.com/24227051/75618172-9d2f3880-5b6a-11ea-8f07-8605b2b5a217.png)

Stays this way if u stick with HW mode
![HWdx11FirstTime](https://user-images.githubusercontent.com/24227051/75618177-b9cb7080-5b6a-11ea-97ca-1dfe5870a662.png)

When u switch to SW mode the speed is still fine (F9)

![HWswitchedtoSW](https://user-images.githubusercontent.com/24227051/75618185-eaaba580-5b6a-11ea-81f2-0c36c16a7610.png)

When u switch back from SW mode to HW mode (F9) the FPS is capped to less than full speed.

![SWswitchedtoHWSecondTime](https://user-images.githubusercontent.com/24227051/75618191-04e58380-5b6b-11ea-8185-ec46fdd541db.png)

I'm not sure my PC is at fault but i haven't had these issues with other games.

This problem is more obvious with the fixes from between 2019-2020.

***PC specifications:***
i7-9750h, 2060m, 32GB Ram with Windows 10 Pro x64

**Comment by vsub** [comment]
nVidia is adding more and more things to the drivers to make the life of emulators users on laptops more difficult
Before(and it's still is)was the Whisper mode which set the FPS to max 40 for all programs that are not in their database so you have to disable it(at least for PCSX2)to get more than 40fps
Then recently they added a fps limiter which I think it is set to off by default

Download nVidia Inspector and run it
Type PCSX2 and hit enter
Go all the way down and you will see Whisper Mode which is set to 0x10000028(which means on and the option below it specify the fps limit)
If the option below is set to 0,and whisper mode is on,the fps will be limited to your refresh rate
If  Whisper mode is set to 0(off),you will get the max fps you can get

When I disable Whisper Mode and make sure the "Frame Rate Limiter" is set to 0,I get the max speed I can possibly get and fps is not capped no matter how many times I switch between SW and HW

You cannot disable Whisper on per program\game from the nvidia control panel,you can either disable it completely for everything or increase the fps limit to up to 120fps but you can disable it with nVidia Inspector

**Comment by RedDevilus** [comment]
Ok you are right it was Whisper mode.
However i have just increase to 120FPS on a per program/game from nvidia control panel for only pcsx2 which is the only program with bad side-effects i've seen.

Weirdest thing is that i play all my games on 60+ for my 144hz screen (steam games) and it never capped. Even other pcsx2 games run mostly fine to full speed (direct 3d11).

This also fixed my problem with SW+HW mode of OpenGL. I haven't used Nvidia inspector yet but i could use that in the future.

Thanks =D

tl;dr very specific case of NVIDIA whisper mode capped to 40 FPS that only affected pcsx2.
